packages/yum/test.sh

```diff
@@ -72,6 +72,7 @@ if [ "${run_test}" = "yes" ]; then
   if [ ${os} != "amazon-linux" ]; then
     ${DNF} install -y redhat-rpm-config
   fi
+  gem install rubygems-requirements-system
```

Comment:
We don't use gem red-arrow here, so I think we don't need here.

Reply:
I added `rubygems-requirements-system` because there should be some gems that work effectively.
I also thought it would be easier to understand since the process would be similar to `apt/test.sh`.